src/lib/dictionaries.ts

```diff
@@ -0,0 +1,9 @@
+import "server-only";
```

Comment:
What is it ? Will it do ? @khanhdan-dev

Reply:
Well, I forgot to install this package. Actually, I used this package to make sure server data fetching functions are never used on the client. 

So why we should care about `client side` in this stuff? The answer is to use `Promise` (async await) for `getDictionary` function that would get current locales' json in server, then make sure the client can show exactly the language text at the first rendering.

Read more: [Internationalization](https://nextjs.org/docs/app/building-your-application/routing/internationalization)

Reply:
just updated the package.